src/main/java/uk/nhs/tis/trainee/notifications/service/NotificationService.java

```diff
@@ -290,35 +300,33 @@ private UserDetails getTraineeDetails(String personId) {
   }
 
   /**
-   * Check whether an object meets the selected notification criteria.
+   * Check whether a programme membership meets the selected notification criteria.
    *
    * @param programmeMembership The programme membership to check.
    * @param checkNewStarter     Whether the trainee must be a new starter.
    * @param checkPilot          Whether the trainee must be in a pilot.
-   * @return true if all criteria bet, or false if one or more criteria fail.
+   * @return true if all criteria met, or false if one or more criteria fail.
    */
-  public boolean meetsCriteria(ProgrammeMembership programmeMembership,
+  public boolean programmeMembershipMeetsCriteria(ProgrammeMembership programmeMembership,
```

Comment:
I'd prefer the original naming since it would make overloads for placement (any whatever else) more easily identified.

Reply:
Cool, I hadn't considered that. Reverted.